### PR TITLE
add github-api-put.spec.js

### PR DESCRIPTION
**Pull request by renardete**

### CODE REVIEW

File: test/github-api-put.spec.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      expect(response.statusCode).toBe(statusCode.OK);
+      expect(followedUser.login).toBe(followedName);
+      expect(followedUser.id).toBe(followedId);
+    });
+  });
+
+  describe('when https://api.github.com/user/following/aperdomob resource is calles', () => {
+    const followUrl = `${urlBase}/user/following/aperdomob`;
+    it('then the response should contain status-code = 204 and the body must be empty', async () => {
+      const response1 = await agent
```
Comment (leonleo997) [inline test/github-api-put.spec.js]: You should choose a better name for this const

File: test/github-api-put.spec.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        .put(followUrl)
+        .auth('token', process.env.ACCESS_TOKEN)
+        .set('User-Agent', 'agent');
+
+      const response2 = await agent
```
Comment (leonleo997) [inline test/github-api-put.spec.js]: You should choose a better name for this const

File: test/github-api-put.spec.js (hunk 1 above)
Comment (leonleo997) [reply test/github-api-put.spec.js]: Add these responses to a before statement

File: test/github-api-put.spec.js (hunk 2 above)
Comment (renardete) [reply test/github-api-put.spec.js]: listo

**Pull request merged**